src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,15 @@
+
+$duration: 500ms;
+$boxSize: 100px;
+$gap: 1px;
+$calendarPadding:10px;
+$fullSize: $boxSize + $gap;
+
+$weekDays :
+"mon" 0,
+"tue" ($fullSize),
+"wed" ($fullSize * 2),
+"thu" ($fullSize * 3),
+"fri" ($fullSize * 4),
+"sat" ($fullSize * 5),
+"sun" ($fullSize * 6);
```

Comment:
you can calculate it in loop

Reply:
How to implement it? I have no idea. 

> you can calculate it in loop

Reply:
I implemented it somehow, but not sure about its validation.